core/src/main/java/tech/pegasys/web3signer/core/Runner.java

```diff
@@ -137,7 +152,10 @@ private VertxOptions createVertxOptions(final MetricsSystem metricsSystem) {
                 .setFactory(new VertxMetricsAdapterFactory(metricsSystem)));
   }
 
-  protected abstract Router populateRouter(final Context context);
+  protected abstract ArtifactSignerProvider getArtifactSignerProvider(final Context context);
```

Comment:
can this be renamed to createArtifactSignerProvider since this isn't really a getter it's a factory method.